### ISSUE TITLE
Y_Hooks not called with 3.10.11 Compiler and Open.MP

### ISSUE THREAD

**Issue by AthallahDzaki** [pawn-lang/YSI-Includes#670]
If i use 3.10.10 Compiler, y_hooks work perfectly.
with 3.10.11 (64 Bit) compiler, hook not call at all only public function.

Hook Function is OnPlayerEnterDynamicRaceCP -> OnPlayerEnterDynRaceCP (x) -> OnPlayerEnterDynamicRaceCP (x)

Note : (x) is not work

![3.10.10](https://github.com/pawn-lang/YSI-Includes/assets/44716687/4d63f080-ce72-41d9-a5e8-e2d6c8069510) 
3.10.10 Log (Called 2 Times)

![3.10.11](https://github.com/pawn-lang/YSI-Includes/assets/44716687/cf983efd-80ff-4593-a03f-fda87e831bcc) 
3.10.11 Log (Called 1 Times)

**Comment by AthallahDzaki** [comment]
Only happend with 31+ Function name

**Comment by Y-Less** [comment]
Let me check that specific case, because y_hooks works in open.mp in general.  There could be something with the double replacements.

**Comment by eksqtr** [comment]
has this been fixed? I'm encountering this issue

**Comment by AthallahDzaki** [comment]
> has this been fixed? I'm encountering this issue

Yes, as i remember, you need to add public FunctionName like public OnPlayerEnterDynamicCP

**Comment by savvin-dev** [comment]
`DEFINE_HOOK_REPLACEMENT__(DynamicRaceCP, DynamicRaceCP);`
